node/clients/search-resolver.ts

```diff
@@ -3,7 +3,13 @@ import { pathOr } from "ramda";
 
 export class SearchResolver extends AppGraphQLClient {
   constructor(context: IOContext, options?: InstanceOptions) {
-    super("vtex.search-resolver", context, options);
+    super("vtex.search-resolver", context, {
+      ...options,
+      headers: {
+        ...(options?.headers ?? {}),
+        'x-vtex-provider': 'vtex.search-graphql'
```

Comment:
I think this won't suffice for updating the schema in runtime, no? Since you're not passing the complete `appId` the resolver won't be able to update its graphql schema at runtime, WDYT?

Reply:
Is it possible to change the implementation and call `vtex.search-graphql` instead of the resolver? I think that if you do it, graphql-server will take care of adding the appropriate headers.

Reply:
Hey I added the full appId to the header. `vtex.search-graphql@0.43.0`. It is updating the schema properly now.